core/src/worker/api.ts

```diff
@@ -1,14 +1,15 @@
 import axios from 'axios'
 import { URL } from 'node:url'
 import { Logger } from 'pino'
-import { IAggregator, IAggregate, IErrorMsgData } from '../types'
+import { IAggregator, IAggregate, IErrorMsgData, IL2AggregatorPair } from '../types'
 import { OraklError, OraklErrorCode } from '../errors'
 import { ORAKL_NETWORK_API_URL } from '../settings'
 import { buildUrl } from '../utils'
 
 export const AGGREGATE_ENDPOINT = buildUrl(ORAKL_NETWORK_API_URL, 'aggregate')
 export const AGGREGATOR_ENDPOINT = buildUrl(ORAKL_NETWORK_API_URL, 'aggregator')
 export const ERROR_ENDPOINT = buildUrl(ORAKL_NETWORK_API_URL, 'error')
+export const L2AGGREGATOR_ENDPOINT = buildUrl(ORAKL_NETWORK_API_URL, 'l2aggregator')
```

Comment:
Readability suggestion.

```suggestion
export const L2_AGGREGATOR_ENDPOINT = buildUrl(ORAKL_NETWORK_API_URL, 'l2aggregator')
```